src/components/quantity-lessons-card/QuantityLessonsChart.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useMemo } from 'react'
+import { Bar } from 'react-chartjs-2'
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  BarElement,
+  Title,
+  Tooltip,
+  Legend
+} from 'chart.js'
+import { Box } from '@mui/material'
+import useBreakpoints from '~/hooks/use-breakpoints'
+
+ChartJS.register(CategoryScale, LinearScale, BarElement, Title, Tooltip, Legend)
+
+const data = {
+  labels: [
+    'JAN',
```

Comment:
its probably better to use Date here

Reply:
It is not needed because in the end result we will get the same results as with Date. We simply have a fixed list of months that doesn't change in any way and doesn't depend on anything.